src/library/Cardano/CoinSelection/Migration.hs

```diff
@@ -184,12 +182,10 @@ selectCoins feeOpts batchSize utxo =
         put rest
         pure batch
 
---------------------------------------------------------------------------------
--- Internal Functions
---------------------------------------------------------------------------------
-
--- Try to find a fixed "ideal" number of input transactions that would generate
+-- | Try to find a fixed "ideal" number of input transactions that would generate
 -- relatively balanced transactions.
```

Comment:
I have a couple of suggestions here. We could:

1. Add a definition of what it means to for a transaction to be "relatively-balanced"? (Justification: A new library user might not even know what a "balanced" transaction is.)

2. Add an explanation of why a user might want to use this function. (Of course, in the wallet, we use it to find a batch size that we then pass to the migration coin selection function, but perhaps this isn't very obvious to the user.)